debian/changelog

```diff
@@ -1,4 +1,4 @@
-amazon-ecr-credential-helper (0.2.0-2) UNRELEASED; urgency=medium
+amazon-ecr-credential-helper (0.3.1-1) UNRELEASED; urgency=low
```

Comment:
There should be an entry explicitly noting that this is a new upstream version. You don't need to go into any detail about what has actually changed, though if the new upstream version fixes any bugs that are open in Debian's BTS then this is where you'd indicate that. [Example](https://tracker.debian.org/media/packages/s/spamassassin/changelog-3.4.3-1)

Reply:
I need to include something like "New upstream release"?  What about the "urgency" level?

Reply:
Done

Reply:
Urgency isn't really meaningful here.